docs/source/top_level_elements/name.rst

```diff
@@ -0,0 +1,350 @@
+
+    [None, 'American', 'Argentine', 'Austrian', 'Brazilian', 'British', 'Canadian', 'Danish', 'Dutch', 'Finnish', 'Flemish', 'French', 'German', 'Greek', 'Italian', 'Japanese', 'Norwegian', 'Spanish', 'Swiss', 'Turkish']
+
+An example is `archivision:4817 <https://digital.lib.utk.edu/collections/islandora/object/archivision%3A4817>`_.
+
+If we need to keep this data, we need to have an external triple store and decide how to model names.
+
+Name Types
+----------
+
+We have 2,297 records with a name with a type attribute equal to corporate and 12,806 records with a name with a type equal
+to personal.
+
+Here are examples of each:
+
+* `egypt:24 <https://digital.lib.utk.edu/collections/islandora/object/egypt%3A24/datastream/MODS>`_
+* `cDanielCartoon:1179 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon%3A1178/datastream/MODS>`_
+
+I don't think we need this information.
+
+Name Display Forms
+------------------
+
+We have 10,284 records with a name/displayForm.
+
+An example is `archivision:2480 <https://digital.lib.utk.edu/collections/islandora/object/archivision%3A2480/datastream/MODS>`_.
+
+.. code-block:: xml
+
+    <name type="personal" authority="ulan" valueURI="http://vocab.getty.edu/ulan/500023721">
+        <namePart>Loos, Adolf</namePart>
+        <displayForm>Adolf Loos</displayForm>
+        <namePart type="date">1870-1933</namePart>
+        <description>Austrian</description>
+        <role>
+            <roleTerm type="text" authority="marcrelator" valueURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+Do we need this?  What are the use cases?
+
+Role Term Types
+---------------
+
+We have 23,811 role terms with a type="text".
+
+An example is `cDanielCartoon:1178 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon%3A1178/datastream/MODS>`_.
+
+.. code-block:: xml
+
+    <name type="personal">
+        <namePart>Daniel, Charles R. (Charlie), Jr., 1930-</namePart>
+        <role>
+            <roleTerm type="text" authority="marcrelator" valueURI=" http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+We don't need to keep this.
+
+Role Term Authority URIs
+------------------------
+
+We have 745 role terms with authorityURIs.
+
+An example is `insurancea:163 <https://digital.lib.utk.edu/collections/islandora/object/insurancena%3A163/datastream/MODS>`_.
+
+.. code-block:: xml
+
+    <name type="personal">
+        <namePart>George Meade</namePart>
+        <role>
+            <roleTerm authority="marcrelator" authorityURI="http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+We don't need to keep this.
+
+Other Xpaths to Trash
+---------------------
+
+There are a few other garbage that we need to trash.  Rather than talking about them in depth, here are the xpaths:
+
+* :code:`name/role/roleTerm/@authority`
+* :code:`name/@authority`
```

Comment:
Do all of these values actually have a valueURI associated with them? If not, this may be something to consider for cleanup. I'd agree that we don't need this going forward.

Reply:
Why would we consider cleaning this up if we're not going to migrate it? Not sure I'm following.

Reply:
Getting this information would let me know automatically if there is a controlled string for the name. I wouldn't migrate the authority information (authority="naf"), but it might let me know that there are a number of names that we can use a valueURI for rather than a string. It would require some effort, but maybe not tons depending on the extent of names like this.